version.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+cd "$(dirname "$0")"
+NAME=$(basename $PWD)
+SOURCE=$(git config remote.origin.url | sed -e 's|git@|https://|g' | sed -e 's|github.com:|github.com/|g' | sed 's|\.git||g')
```

Comment:
Oh I think I get it: ssh vs. https. Then no need to strip the trailing `.git` https://shavar.services.mozilla.com/__version__